### PR TITLE
[doc] add presto doc

### LINKED ISSUE #667: [Feature]  Introduce Presto Reader for Paimon
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/incubator-paimon/issues) and found nothing similar.

### Motivation

Introduce Presto Reader for Paimon

### Solution

_No response_

### Anything else?

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by s7monk**
fix #667

### Purpose

add presto doc

### Tests

no

### API and Format 

no

### Documentation

engines#presto.md engines#overview.md project#download.md

### CODE REVIEW

File: docs/content/engines/presto.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+If you are using HDFS, choose one of the following ways to configure your HDFS:
+
+- set environment variable HADOOP_HOME.
+- set environment variable HADOOP_CONF_DIR.
+- configure fs.hdfs.hadoopconf in the properties.
+
+You can configure kerberos keytag file when using KERBEROS authentication in the properties.
+
+```
+security.kerberos.login.principal=hadoop-user
```
Comment (JingsongLi) [inline docs/content/engines/presto.md:89]: Dose `paimon-presto` invoke `SecurityContext.install?`

Comment (s7monk) [reply docs/content/engines/presto.md:89]: currently not calling

Comment (JingsongLi) [reply docs/content/engines/presto.md:89]: so maybe this is not work now.

Comment (s7monk) [reply docs/content/engines/presto.md:89]: yes, i'll do it later

Comment (s7monk) [reply docs/content/engines/presto.md:89]: Added SecurityContext.install @JingsongLi

**Review by JingsongLi — APPROVED** [review]
Looks good to me!

**Pull request merged**